development/kops/prow.sh

```diff
@@ -13,33 +13,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ "$PROW_ROLE_ARN" == "" ]; then
-    echo "Empty PROW_ROLE_ARN, this script must be run in a postsubmit pod with IAM Roles for Service Accounts"
+if [ "$AWS_ROLE_ARN" == "" ]; then
```

Comment:
Maybe call this SOURCE_ROLE_ARN?

Reply:
I just used the same name we are using for docs.